openfisca_web_api/loader/parameters.py

```diff
@@ -95,17 +95,21 @@ def build_parameters(tax_benefit_system, country_package_metadata):
     api_parameters = []
     walk_node(
         original_parameters,
-        parameters = api_parameters,
-        path_fragments = [],
-        country_package_metadata = country_package_metadata,
+        parameters=api_parameters,
```

Comment:
The codebase doesn't follow this PEP recommandation (see [recent debates [FR]](https://github.com/openfisca/openfisca-france/pull/1060#issuecomment-410064151)). It's explicitly ignored in the `setup.cfg` file. 

For consistency, I'd prefer keeping the spaces 🙂